browser/lib/transport/xhrrequest.js

```diff
@@ -122,19 +122,19 @@ var XHRRequest = (function() {
 		for(var h in headers)
 			xhr.setRequestHeader(h, headers[h]);
 
-		var onerror = xhr.onerror = function(err) {
-			err.code = 80000;
-			self.complete(err);
+		var errorHandler = function(errorEvent, message, code, statusCode) {
+			var errorMessage = message + ", errorEvent was " + Utils.inspect(errorEvent) + ", current statusText is " + self.xhr.statusText;
```

Comment:
> but I figure leaving it there doesn't cost much. Want me to remove it?

Looking at the event specs you're not going to get anything useful from the event object itself except the type of the event (ie error or abort). TBH I' remove the `inspect()` and just have `'type = ' + ev.type` or something like that in the log message.